fighters/link/src/acmd/aerials.rs

```diff
@@ -258,13 +224,37 @@ unsafe fn attack_air_lw(fighter: &mut L2CAgentBase) {
     
 }
 
+#[acmd_script( agent = "link", script = "effect_attackairlw" , category = ACMD_EFFECT , low_priority)]
+unsafe fn effect_attackairlw (fighter: &mut L2CAgentBase) {
```

Comment:
Sword glow gfx is missing ingame

Reply:
Returned link_sword_flare effect size from 0.3 to 1.0